README.md

```diff
@@ -1,4 +1,54 @@
 # Dcrtimestamptweet
 
-> Twitter bot to timestamp a thread when @dcrtimestamptweet is mentioned.
+Twitter bot to timestamp a thread when @dcrtimestamptweet is mentioned. This bot uses IPFS as a decentralized database to store the tweet thread. 
 
+## Installation
+
+1. Make a twitter development app and generate tokens: https://developer.twitter.com/en/docs/basics/authentication/guides/access-tokens.html
+
+2. Make a .env File
+    ```
+    $ touch .env
+    ```
+
+3. Add the generated tokens
+    ```
+    TWITTER_CONSUMER_KEY=<GENERATED_API_KEY>
+    TWITTER_CONSUMER_SECRET=<GENERATED_API_KEY_SECRET>
+    TWITTER_ACCESS_TOKEN=<GENERATED_ACCESS_TOKEN>
+    TWITTER_ACCESS_TOKEN_SECRET=<GENERATED_ACCESS_TOKEN_SECRET>
+    TRACKED_WORD="@dcrtimestamptweet" // mention which will call the bot
+    ```
+
+4. Install IPFS
+    
+    4.1 IPFS Install instructions: https://docs.ipfs.io/introduction/install/
+
+5. Run IPFS Daemon 
+    ```
+    ipfs daemon
+    ```
+
+6. Run yarn command
+    ```
+    $ yarn
+    ```
+
+7. Start app
+    ```
+    yarn start
+    ```
+
+Now when TRACKED_WORD is mentioned on some tweet the bot will save the thread at the ipfs and timestamp it and will tweet the hashes.
+
+IPFS JSON generated
```

Comment:
I don't think this example is necessary here.

Reply:
IMO we can keep the example so people have a clue of what properties from twitter we are timestamping.